client/js/widgets/widget.js

```diff
@@ -519,7 +519,11 @@ export class Widget extends StateManaged {
               const parent = clone.parent;
               clone.clonedFrom = w.get('id');
 
-              delete clone.id;
+              if(widgets.has(clone.id)) {
+                delete clone.id;
+                if(properties.has('id'))
```

Comment:
Does this work? It's just an object and should be `properties.id !== undefined`, right?

Reply:
Yes. I was working with Set earlier and was thinking of that interface.